functions/common/debezium-autoconfigure/README.adoc

```diff
@@ -0,0 +1,280 @@
+# Debezium Auto-Configuration
+
+This module provides a generic https://debezium.io/documentation/reference/2.2/development/engine.html[DebeziumEngine.Builder] auto-configuration that can be reused and composed in other applications.
+
+NOTE: The `DebeziumEngine` doesn't not required Kafka or Kafka Connect as it runs embedded inside your application. This approach though comes with some delivery guarantee limitations as explained https://debezium.io/documentation/reference/2.2/development/engine.html#_handling_failures[here].
+
+The `Debezium Engine` is a https://en.wikipedia.org/wiki/Change_data_capture[Change Data Capture] (CDC) utility, that allows *capturing* database change events and process them with custom `java.util.Consumer` or `io.debezium.engine.ChangeConsumer` event handler implementations.
+
+The `DebeziumEngine.Builder` auto-configuration is activated only if a https://debezium.io/documentation/reference/stable/connectors/index.html[Debezium Connector] is found on the classpath and the `debezium.properties.connector.class` property is set to point to that connector class.
+
+## Quick Start
+
+To use of the `DebeziumEngine` one need to (1) add required <<dependencies>> to the project, (2) provide the necessary Debezium <<configuration-options>>, (3) implement custom <<changeEvent-handlers>> and register it with the engine and (4) build and submit the DebeziumEngine to an `Executor` or `ExecutorService` for execution.
+
+[[dependencies]]
+### Dependencies
+
+To process the incoming database change events you need to include the Debezium `auto-configuration` dependency to your project:
+
+====
+[source, xml, subs="normal", role="primary"]
+.Maven
+----
+<dependency>
+    <groupId>org.springframework.cloud.fn</groupId>
+    <artifactId>debezium-autoconfigure</artifactId>
+    <version>{project-version}</version>
```

Comment:
I think this should be `${project.version}`.

Reply:
I wonder if we can inject variables with the real versions here

Reply:
We could always add this in subsequent PR

Reply:
This is not possible.
See: this is a top-level project `README.adoc` which is fully static, unlike all those variants you suggest for docs generation.